DM-19890: Auxiliary Telescope in-dome software test

tribeiro opened the pull request:
This PR adds the notebooks developed during the visit to AT in 2019/05.

tribeiro committed:
Some changes to Bokeh Test notebook.

tribeiro committed:
Add nostebooks for ATTracking test.

tribeiro committed:
Add missing notebooks.

tribeiro committed:
Add notebooks from AT mission 2019/05

tribeiro committed:
Latest changes and notebooks.

couger01 commented on tribeiro/Untitled.ipynb:
```diff
@@ -0,0 +1,168 @@
+ },
+ "nbformat": 4,
+ "nbformat_minor": 2
+}
```
Could you title this?

couger01 commented on tribeiro/Untitled1.ipynb:
```diff
@@ -0,0 +1,101 @@
+ },
+ "nbformat": 4,
+ "nbformat_minor": 2
+}
```
Could you title this?

couger01 commented on tribeiro/Untitled2.ipynb:
```diff
@@ -0,0 +1,50 @@
+ },
+ "nbformat": 4,
+ "nbformat_minor": 2
+}
```
Could you title this?

couger01 reviewed (changes requested):
Looks good to me, but I'd like the untitled notebooks to have some titles.

tribeiro committed:
Remove useless Untitled notebooks and add notebook to measure m1 pressure from dataset.

tribeiro commented:
Thanks for spotting those. I removed then because they where pretty much useless and added a new one with measurement of the M1 pressure coefficients from the dataset.

couger01 reviewed (approved):
LGTM.
